packages/react/src/components/pagination/pagination.template.tsx

```diff
@@ -0,0 +1,15 @@
+import { Pagination } from '@dso-toolkit/sources';
+import * as React from 'react';
+
+import { DsoPagination } from '../..';
+
+export function paginationTemplate({ totalPages, currentPage, onSelectPage, formatHref }: Pagination) {
+  return (
+    <DsoPagination
+      totalPages={totalPages}
+      currentPage={currentPage}
+      formatHref={formatHref}
+      selectPage={onSelectPage}
```

Comment:
De React bindings van Stencil prefixen `@Event()`s met `on`: Deze property zou volgens mij `onSelectPage` moeten zijn, want de events tonen niet in Storybook: https://react.dso-toolkit.nl/_1638-pagination-web-component/?path=/story/pagination--pagination